code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -111,3 +111,38 @@ Lizard subspecies: ASHWALKERS
 	species_traits = list(MUTCOLORS,EYECOLOR,LIPS,DIGITIGRADE,HAS_FLESH,HAS_BONE)
 	inherent_traits = list(TRAIT_ADVANCEDTOOLUSER,TRAIT_CHUNKYFINGERS,TRAIT_NOBREATH)
 	species_language_holder = /datum/language_holder/lizard/ash
+
+/*
+Lizard subspecies: SILVER SCALED
+*/
+/datum/species/lizard/silverscale
+	name = "Silver Scale"
+	id = "silverlizard"
+	limbs_id = "lizard"
+	inherent_biotypes = MOB_ORGANIC|MOB_HUMANOID|MOB_REPTILE
+	inherent_traits = list(TRAIT_ADVANCEDTOOLUSER,TRAIT_HOLY,TRAIT_NOBREATH,TRAIT_VIRUSIMMUNE,TRAIT_WINE_TASTER)
+	species_language_holder = /datum/language_holder/lizard/silver
+	mutanttongue = /obj/item/organ/tongue/lizard/silver
+	armor = 10 //very light silvery scales soften blows
+	changesource_flags = MIRROR_BADMIN | MIRROR_MAGIC | RACE_SWAP | ERT_SPAWN
```

Comment:
let people choose this from the pride mirror too

Reply:
No thanks